test/knative-addon-validation.test.ts

```diff
@@ -0,0 +1,20 @@
+import * as cdk from 'aws-cdk-lib';
+import * as blueprints from '../lib';
+import { IstioBaseAddOn, IstioControlPlaneAddOn, KNativeOperator } from "../lib";
+
+test("Generic cluster with kNative Eventing deployment", async () => {
+    const app = new cdk.App();
+
+    const addons = [
+        new IstioBaseAddOn(),
+        new IstioControlPlaneAddOn(),
+        new KNativeOperator(),
+    ];
+
+    const blueprint = await blueprints.EksBlueprint.builder()
+        .account('123456789').region('us-east-1')
+        .addOns(...addons)
+        .build(app, 'knative-stack');
+
+    expect(blueprint).toBeDefined();
```

Comment:
Blueprint will be defined no matter what. How about expecting blueprint to have a resource of a particular type with the name knative? At least it will protect from someone deleting the body of the addon (do nothing on deploy).